test/hapi_server_v17.js

```diff
@@ -0,0 +1,115 @@
+const EventLogger = require('../').EventLogger;
```

Comment:
Missing error tests? Or were they excluded on purpose?

Reply:
Not sure if i follow; what do you mean?